<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>471</width>
      <height>361</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Ancient One</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>451</width>
        <height>131</height>
       </rect>
      </property>
      <property name="title">
       <string>Worshipers</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>431</width>
         <height>71</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Description</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="txtAncientOncesWorshipersDescription"/>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>12</x>
         <y>22</y>
         <width>23</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Type</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtAncientOneWorshipersType">
       <property name="geometry">
        <rect>
         <x>41</x>
         <y>22</y>
         <width>181</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>229</x>
         <y>22</y>
         <width>25</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Mods</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtAncientOneWorshipersMods">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>22</y>
         <width>181</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>0</y>
        <width>231</width>
        <height>191</height>
       </rect>
      </property>
      <property name="title">
       <string>Power</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>211</width>
         <height>131</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="lvlAncientOncePowerDescription">
          <property name="text">
           <string>Description</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="txtAncientOnePowerDescription"/>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="lblAncientOncePowerName">
       <property name="geometry">
        <rect>
         <x>12</x>
         <y>22</y>
         <width>27</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtAncientOncePowerName">
       <property name="geometry">
        <rect>
         <x>45</x>
         <y>22</y>
         <width>171</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>12</y>
        <width>212</width>
        <height>181</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="lblAncientOneName">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2" colspan="4">
        <widget class="QLineEdit" name="txtAncientOneName"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblAncientOneTitle">
         <property name="text">
          <string>Title</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="5">
        <widget class="QLineEdit" name="txtAncientOneTitle"/>
       </item>
       <item row="2" column="0" colspan="5">
        <widget class="QLabel" name="lblAncientOneCombatRating">
         <property name="text">
          <string>Combat Rating</string>
         </property>
        </widget>
       </item>
       <item row="2" column="5">
        <widget class="QLineEdit" name="txtAncientOneCombatRating"/>
       </item>
       <item row="3" column="0" colspan="4">
        <widget class="QLabel" name="lblAncientOneDoomTrack">
         <property name="text">
          <string>Doom Track</string>
         </property>
        </widget>
       </item>
       <item row="3" column="4" colspan="2">
        <widget class="QLineEdit" name="txtAncientOneDoomTrack"/>
       </item>
       <item row="4" column="0" colspan="3">
        <widget class="QLabel" name="lblAncientOneDefence">
         <property name="text">
          <string>Defence</string>
         </property>
        </widget>
       </item>
       <item row="4" column="3" colspan="3">
        <widget class="QLineEdit" name="txtAncientOneDefence"/>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QLabel" name="lblAncientOneAttack">
         <property name="text">
          <string>Attack</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2" colspan="4">
        <widget class="QLineEdit" name="txtAncientOneAttack"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>investigator</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>380</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Click to save XML data</string>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
